src/Chapter1.hs

```diff
@@ -631,8 +641,15 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
+sumLast2 :: Int -> Int 
+sumLast2 n = last n + penultimate n
+  where 
+    last :: Int -> Int
+    last n = (abs n) `mod` 10 
```

Comment:
`last` is the name of a standard function as well, so it's better to rename this one 🙂 Though you can notice that it is actually the `lastDigit` function from the previous task 🙂